enhancements/console/internationalization.md

```diff
@@ -110,6 +110,8 @@ The desired fallback experience if not all pieces/parts of the UI are translated
 
 We will also modify our existing file conversion tools to handle translation files spread across multiple packages so we can easily export/import translation files.
 
+Translation of the login templates within [oauth-templates](https://github.com/openshift/oauth-templates) and [oauth-server](https://github.com/openshift/oauth-server) repositories will be manually updated by the console team since their are only a small number of strings that rarely change. These strings are to be included in the console repository so that the i18next-parser will add them in the files sent to the Globalization team for translation.
```

Comment:
We should add more detail here like how we are using the `Accept-Language` header and optional `lng` query parameter to determine what language to show on the backend. We might want @jhadvig to contribute that detail since he did the backend work. We should also discuss if/how we plan to handle the language user preference for the login page. (See discussion in https://github.com/openshift/oauth-server/pull/71)

Reply:
@spadgett The text has been updated with additional detail provided by @jhadvig